db/migrate/20170809122812_add_created_at_to_awarded_point.rb

```diff
@@ -0,0 +1,8 @@
+class AddCreatedAtToAwardedPoint < ActiveRecord::Migration
+  def change
+    add_column :awarded_points, :created_at, :datetime
+    AwardedPoint.find_each do |awarded_point|
+      awarded_point.update!(created_at: awarded_point.submission.created_at)
```

Comment:
Actually submissions can be deleted and many for many awarded points submission is actually null. So this wont work on our production server :stuck_out_tongue:.

Maybe we should change this to:
```ruby
awarded_point.update!(created_at: awarded_point.submission.created_at) if awarded_point.submission
```